lib/datastore/request.js

```diff
@@ -821,8 +854,9 @@ DatastoreRequest.prototype.request_ = function(protoOpts, reqOpts, callback) {
   }
 
   if (isTransaction && (method === 'lookup' || method === 'runQuery')) {
```

Comment:
This will overwrite the read options. Maybe you should make it an error if isTransaction and readOptions are both set?